keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/ApplicationControllerTests.kt

```diff
@@ -171,17 +171,17 @@ internal class ApplicationControllerTests : JUnit5Minutests {
         deliveryConfig.environments.flatMap { it.resources }.forEach { resource ->
           resourceRepository.appendHistory(ResourceValid(resource))
         }
```

Comment:
Using actual app versions here makes the tests harder to read, but ensures that we test important string parsing for debians.

Reply:
Funny enough, I had come across this exact problem earlier, and added proper version strings, but forgot to port those over when I rewrote the PR. https://github.com/spinnaker/keel/pull/883/files#diff-fc6833fb02ee833fc028c7cdb6c7037a

Sorry to give you unnecessary work. 🙁